hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -162,16 +166,18 @@
         // '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
         //        -o hw/top_earlgrey --rnd_cnst_seed {seed}
         // ''',
+        // Regenerate entropy buffer files (from a seed) used by the
+        // `gen-lc-state-enc.py`, `gen-otp-mmap.py`, and `gen-otp-img.py` scripts.
+	      '''{gen_entropy_buffer_cmd} {gen_entropy_buffer_cmd_opts} \
+               --seed {seed} -o {lc_state_enc_entropy_file_path}
+        ''',
+	      '''{gen_entropy_buffer_cmd} {gen_entropy_buffer_cmd_opts} \
+               --seed {seed} -o {otp_mmap_entropy_file_path}
+        ''',
```

Comment:
I think that this would generate two identical entropy buffer files.
In the old flow it was OK to use the same seed, because this seed was passed down to `LcStEnc.py` and `OtpMemMap.py` where diversifiers were added before seeding the PRNG.

I'm not exactly sure how to fix this. Would it be useful to add `--div {diversifier}` option to `entropy_buffer_generator.py` to create the entropy buffer file using seed `{seed} + {diversifier}' ?